### ISSUE TITLE
JetReview - Review Types for Admins

### ISSUE THREAD

**Issue by hb81010** [CrocoBlock/suggestions#5152]
I have some websites where I test and review some products myself. Now unfortunately JetReview only allows you to create "Review Types" as review templates for website visitors.

It would be nice if the created review types could also be made available to the admin in the dashboard (similar to the meta fields), so that they could be rated there and inserted as a widget in the template for the product page for example.

I tried the Static Widget, but then the ratings are not dynamic and so I can't integrate them into my product's page template.

Currently I am solving the problem with another plugin, however JetReview is much better in my opinion, just unfortunately I am missing this important option to create dynamic admin product reviews on my site.

Maybe you could integrate that. Especially in the area of websites that test products, there are very few solutions and none is really 100% satisfied.

Here is an example of the current plugin (don't know if I'm allowed to name it here) that I use. My created review criteria incl. fields for pros and cons. The associated Elementor widget of the plugin gets into the product-template and then the rating is displayed.

![68747470733a2f2f756c74696d6174652d72657669657765722e67686f7374706f6f6c2e636f6d2f77702d636f6e74656e742f75706c6f6164732f323031382f31312f686f6d652d7265766965772d73657474696e67732e706e67](https://user-images.githubusercontent.com/54399588/162198112-b99c71b8-11f4-4b76-bbc3-873b83ef6bb0.png)

**Comment by shinTM** [comment]
if I understood you correctly https://tppr.me/TBcZF

**Comment by hb81010** [comment]
Yes, that's what I tried first. When I insert the created review type, with the Review Listing Widget in my template, then I have all the rating criteria, but they are rather for the visitors of my site, so that they can rate.

But I'm trying to give my review as a website owner for a product. I have a page on which I test and rate products. The Static Review Widget goes in a similar direction, but unfortunately I only have the option to enter the values manually or to create the ReviewField for each product in the backend (PostMeta). Unfortunately it does not have a consistent scheme and cannot filter the rated products with meta fields or display the overall rating in listings, for example, because it is static.


That's why I thought it would be somehow possible that the created review listing could be displayed in the admin backend instead of this manual part of JetReview.

![grafik](https://user-images.githubusercontent.com/54399588/164768243-f4528c7d-0f0c-4622-b8ef-ebdad3e79448.png)
